### DISCUSSION TITLE
Is there a way to exclude images from being darkened?

### DISCUSSION THREAD

**Discussion by notababyanimal** [darkreader/darkreader#8150] [Q&A]
As per the title - I'm wondering if there's a way to set this extension so that images will not be darkened. This morning the extension suddenly began to dull most images on most sites when in dynamic mode (I'm not sure if this counts as a bug or if there's a way to fix it already baked into the extension, thus why I'm starting a discussion rather than making a bug report) - I switched to static mode and that left the images alone sometimes, but broke other websites somehow. I tried Filter+ as well which didn't seem to make a difference, but I'm also using Firefox which Filter+ apparently works poorly on.

So: is there a toggle somewhere within the extension that will ignore images? I've looked but if there is one, it's not jumping out at me.

**Comment by Gusted** [comment]
#8145

**Reply by notababyanimal** [reply]
Thanks for this! I had skipped over it due to my issue not being inversion, but that seems to have fixed my issue.
